tests/codes/eurocode/nen_en_1993_5_2008/chapter_5_ultimate_limit_states/test_formula_5_9.py

```diff
@@ -0,0 +1,96 @@
+"""Testing formula 5.9 of NEN-EN 1993-5:2008."""
+
+import pytest
+
+from blueprints.codes.eurocode.nen_en_1993_5_2008.chapter_5_ultimate_limit_states.formula_5_9 import Form5Dot9ReducedBendingMomentResistance
+from blueprints.validations import LessOrEqualToZeroError
+
+
+class TestForm5Dot9ReducedBendingMomentResistance:
+    """Validation for formula 5.9 from NEN-EN 1993-5:2008."""
+
+    @pytest.mark.parametrize(
+        ("beta_b", "w_pl", "rho", "a_v", "t_w", "alpha", "f_y", "gamma_m_0", "mc_rd", "expected_value"),
+        [
+            (0.8, 500, 0.25, 100, 10, 45, 250, 1.0, 400, 99.9779),  # Mv,Rd < Mc,Rd
+            (0.8, 500, 0.25, 100, 10, 45, 250, 1.0, 50, 50),  # Mv,Rd < Mc,Rd
+        ],
```

Comment:
why input all values if you are only changing 2 of them? It is much more readable if you keep the constants just as a constant inside the method body.